server/ext_out.c

```diff
@@ -41,6 +41,7 @@ struct ext_out {
 
 
 static struct ext_out *samples_ext_out;
+static char samples_field_name[MAX_NAME_LENGTH];
```

Comment:
I'm confused: don't we already have a mechanism for giving the samples a field name?  After all, sample count is a legitimate selection, thought we were faking that?